### ISSUE TITLE
`mu_r_norm` in the receptor graph representation seems unused in the following training/evaluating/inference processes?

### ISSUE THREAD

**Issue by VicFisher** [gcorso/DiffDock#111]
I am reading the code of the `TensorProductScoreModel` class, where I find that ,in the `forward` function, the previous calculated attribute `complex_graph['receptor'].mu_r_norm`  seems unused.  

Here is the code in the function `get_calpha_graph` of `datasets/process_mols.py` regarding [`complex_graph['receptor'].mu_r_norm`](https://github.com/gcorso/DiffDock/blob/c1abe25540961f2a88ad0e11938243f62b638b3e/datasets/process_mols.py#L328):
`valid_dist_np = distances[i, dst]`
`sigma = np.array([1., 2., 5., 10., 30.]).reshape((-1, 1))`
`weights = softmax(- valid_dist_np.reshape((1, -1)) ** 2 / sigma, axis=1)  # (sigma_num, neigh_num)`
`assert weights[0].sum() > 1 - 1e-2 and weights[0].sum() < 1.01`
`diff_vecs = c_alpha_coords[src, :] - c_alpha_coords[dst, :]  # (neigh_num, 3)`
`mean_vec = weights.dot(diff_vecs)  # (sigma_num, 3)`
`denominator = weights.dot(np.linalg.norm(diff_vecs, axis=1))  # (sigma_num,)`
`mean_vec_ratio_norm = np.linalg.norm(mean_vec, axis=1) / denominator  # (sigma_num,)`
`mean_norm_list.append(mean_vec_ratio_norm)`

`mu_r_norm = torch.from_numpy(np.array(mean_norm_list).astype(np.float32))`
`complex_graph['receptor'].mu_r_norm = mu_r_norm`

I don't know whether it is important or I missed something and didn't find other issues metioned this. 
Any explanations? Thanks!
